plugins/paper/src/main/java/com/playmonumenta/plugins/depths/abilities/steelsage/DepthsSharpshooter.java

```diff
@@ -73,6 +75,7 @@ public static void addStacks(Plugin plugin, Player player, int stacks) {
 		if (ss != null) {
 			ss.mStacks = Math.min(MAX_STACKS, ss.mStacks + stacks);
 			MessagingUtils.sendActionBarMessage(plugin, player, "Sharpshooter Stacks: " + ss.mStacks);
+			ss.mPlugin.mClientModIntegration.updateAbility(ss.mPlayer, ss);
```

Comment:
I know this is the style all over the place, but I'm trying to get away from it. 

Can we make this ClientModIntegration.updateAbility() a static function? Since only one listener will ever exist, it can keep a static reference to itself. That way we don't have to bounce through mPlugin.mWhatever

Reply:
Sure, after all it is already a singleton (just referenced via another singleton).